models/Fields.js

```diff
@@ -18,6 +18,7 @@ const volunteerFields = [
   "individual",
   "organization",
   "ref",
+  "status",
   "createdAt",
```

Comment:
createdAt & status should not be added into the structure we have for saving the whole form

Reply:
I moved createdAt to another queryFields and validation will only look there for search api calls. can you remove createdAt from voluntter and appeals. 
And also move status to queryFields